### DISCUSSION TITLE
Default options in a separate file

### DISCUSSION THREAD

**Discussion by nicolaceliento** [davidstutz/bootstrap-multiselect#1192] [Q&A] [answered]
Hi all,

there is a way to specify default options in a separate file in order to have a defautl behavior of the plugin?

I tryied this but it doesn't work:

```
$.extend(true, $.fn.Multiselect.prototype.defaults, {
    nonSelectedText: 'Not selected',        //Custom translation
    enableCaseInsensitiveFiltering: true    //Custom default behavior
});
```

I hope you can help me.

Thanks in advance.

**Comment by deygordillos** [comment, marked as answer]
Hi.
I did it this way

$.extend(true, $.fn.multiselect.Constructor.prototype.defaults, {
    nonSelectedText: 'Not selected',        //Custom translation
    enableCaseInsensitiveFiltering: true    //Custom default behavior
});
